specification/datasets/cost_and_usage/columns/allocatedresourcedetails.md

```diff
@@ -0,0 +1,120 @@
+# Allocated Resource Details
+
+Allocated Resource Details provides information about how resources are allocated when using split cost allocation.
```

Comment:
nit: Do we need to introduce the term "split cost allocation" here? I worry that's adding complexity with unnecessary jargon. Can we just say something along the lines of "when usage records are split to support cost allocation requirements."?
```suggestion
Allocated Resource Details provides information about how resources are allocated when usage records are split to support cost allocation requirements.
```